src/engine/test/integration_tests/steps/kvdb-steps.py

```diff
@@ -0,0 +1,139 @@
+def step_impl(context,success):
+    if success=='success':
+        assert context.result['data']['status'] == 'OK'
+    elif success=='error':
+        assert context.result['data']['status'] != 'OK'
+
+@given('I have already created a database named "{database_name}" using the KVDB API')
+def step_impl(context, database_name:str):
+    try:
+        context.result = resource_handler._base_send_command_kvdb(DEFAULT_API_SOCK, "post", {"name": database_name})
+    except:
+        raise Exception('STEP: Couldn''t send request to API')
+
+
+@when('I send a {request_type} request with the database name "{database_name}"')
+def step_impl(context, request_type:str, database_name:str):
+    try:
+        context.result = resource_handler._base_send_command_kvdb(DEFAULT_API_SOCK, request_type.lower(), {"name": database_name})
+    except:
+        raise Exception('STEP: Couldn''t send request to API')
+
+
+@then('I should receive an {request_result} response indicating that the database name already exists')
+def step_impl(context, request_result):
+    if request_result=='success':
+        assert context.result['data']['status'] == 'OK'
+    elif request_result=='error':
+        assert context.result['data']['status'] == 'ERROR'
+        assert context.result['data']['error'] == 'The Database already exists.'
+
+@given('I have a database named "{database_name}" created using the KVDB API')
+def step_impl(context, database_name:str):
+    try:
+        context.result = resource_handler._base_send_command_kvdb(DEFAULT_API_SOCK, "post", {"name": database_name})
+    except:
+        raise Exception('STEP: Couldn''t send request to API')
+
+
+@when('I send a {request_type} request to "{database_name}"')
+def step_impl(context, request_type:str, database_name:str):
+    try:
+        context.result = resource_handler._base_send_command_kvdb(DEFAULT_API_SOCK, request_type.lower(), {"name": database_name})
+    except:
+        raise Exception('STEP: Couldn''t send request to API')
+
+
+@then('I should receive a {request_result} response indicating the database "{database_name}" has been deleted')
+def step_impl(context, request_result:str, database_name:str):
+    if request_result=='success':
+        assert context.result['data']['status'] == 'OK'
+    elif request_result=='error':
+        assert context.result['data']['status'] == 'ERROR'
+
+
+@when('I send a {request_type} request to add a key-value pair to the database "{database_name}" with key "{key_name}" and value "{key_value}"')
+def step_impl(context, request_type:str, database_name:str, key_name:str, key_value:str):
+    try:
+        context.result = resource_handler._base_send_command_kvdb(DEFAULT_API_SOCK, request_type.lower(), {"name": database_name, "entry":{"key": key_name, "value":key_value}}, "db")
+    except:
+        raise Exception('STEP: Couldn''t send request to API')
+
+
+@then('I should receive a {request_result} response with the new key-value pair information')
```

Comment:
Could we unify the step of this line with that of line 116, adapting the responses received?